packages/embla-carousel/src/CarouselNextButton.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, type ButtonProps, useIcon } from "@salt-ds/core";
+import { type MouseEventHandler, forwardRef, useCallback } from "react";
+import { usePrevNextButtons } from "./usePrevNextButtons";
+
+/**
+ * Props for the CarouselNextButton component.
+ */
+export interface CarouselNextButtonProps extends Omit<ButtonProps, "variant"> {}
```

Comment:
are we removing variant because its deprecated to avoid it being used for a new component, or do we always want this button to look the same? asking because we are allowing appearance and sentiment, so bordered and neutral below will be overridden if provided

Reply:
https://github.com/jpmorganchase/salt-ds/pull/5185#discussion_r2171634528

Reply:
deprecated, but going to omit a few more props that we can add later if needed (loading etc)